CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to the APM Agent
+
+The APM Agent is open source and we love to receive contributions from our community — you!
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code.
```

Comment:
To make future diffs more useful, we try to add lines breaks after punctuation marks, i.e:

```
There are many ways to contribute,
from writing tutorials or blog posts,
improving the documentation,
submitting bug reports and feature requests or writing code.
```

(just created a PR changing this for the APM Server as well: elastic/apm-server#619)